pkg/hubble/relay/observer/server.go

```diff
@@ -169,8 +172,15 @@ func (s *Server) ServerStatus(ctx context.Context, req *observerpb.ServerStatusR
 				"No connection to peer %s, skipping", p.Name,
 			)
 			s.peers.ReportOffline(p.Name)
+			// we dont want mean to be exhaustive here as reporting all
```

Comment:
if truncated, we should show something like `... and {{42}} more` a the end

Reply:
Yes, I plan to do this when implementing the CLI counterpart of this PR.